apiserver/facades/client/bundle/bundle_test.go

```diff
@@ -8,34 +8,49 @@ import (
 	gc "gopkg.in/check.v1"
 	"gopkg.in/juju/names.v2"
 
+	"github.com/juju/description"
+	"github.com/juju/juju/apiserver/facade"
 	"github.com/juju/juju/apiserver/facades/client/bundle"
 	"github.com/juju/juju/apiserver/params"
 	apiservertesting "github.com/juju/juju/apiserver/testing"
 	coretesting "github.com/juju/juju/testing"
+	"github.com/juju/juju/worker/workertest"
 )
 
 type bundleSuite struct {
 	coretesting.BaseSuite
-	facade bundle.Bundle
+
+	api  *bundle.Facade
+	auth facade.Authorizer
+	st   *mockState
 }
 
 var _ = gc.Suite(&bundleSuite{})
 
 func (s *bundleSuite) SetUpTest(c *gc.C) {
 	s.BaseSuite.SetUpTest(c)
-	auth := apiservertesting.FakeAuthorizer{
+	/*s.auth = apiservertesting.FakeAuthorizer{
 		Tag: names.NewUserTag("who"),
+	}*/
+	s.auth = &apiservertesting.FakeAuthorizer{
+		Tag: names.NewApplicationTag("gitlab"),
```

Comment:
This is a client facade so the auth tag needs to be a user

Reply:
ok